src/App.js

```diff
@@ -37,7 +37,7 @@ class App extends Component {
   }
 
   addFavorite = (newMovie) => {
-    postFavoriteMovie(newMovie)
+    return postFavoriteMovie(newMovie)
```

Comment:
I love that this solution relied on us to return, remember our first fetch call we wrote for this project we completely forgot that functions have to RETURN something?